images/build/Dockerfiles/vso.bullseye.Dockerfile

```diff
@@ -0,0 +1,325 @@
+    && make altinstall \
+    && python3.10 -V \
+    && cp -R /tmp/Python-3.10.4/* /opt/python/3.10.4/bin/ \
+    && cp /opt/python/3.10.4/bin/python /opt/python/3.10.4/bin/python3.10 \
+    && cd $INSTALLATION_DIRECTORY \
+    && ls \
+    && ln -s /opt/python/3.10/bin/python /usr/bin/python \
+    && [ -d "/opt/python/$PYTHON39_VERSION" ] && echo /opt/python/$PYTHON39_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
+    && [ -d "/opt/python/$PYTHON310_VERSION" ] && echo /opt/python/$PYTHON310_VERSION/lib >> /etc/ld.so.conf.d/python.conf \
+    && ldconfig \
+    && cd /opt/python \
+    && ln -s $PYTHON39_VERSION 3.9 \
+    && ln -s $PYTHON310_VERSION 3.10 \
+    && ln -s $PYTHON310_VERSION latest \
+    && ln -s $PYTHON310_VERSION stable \
+    && ln -s $PYTHON310_VERSION 3 \
+    && ln -sfn /opt/python/$PYTHON310_VERSION /home/codespace/.python/current \
+    # Install PHP pre-reqs
+    && $imagesDir/build/php/prereqs/installPrereqs.sh \
+    # new next line
+    #&& add-apt-repository --remove ppa:xapienz/curl34 \
+    && mkdir -p /home/codespace/.php \
+    # Copy PHP versions
+    && . $buildDir/__phpVersions.sh \
+    && $imagesDir/installPlatform.sh php $PHP80_VERSION \
+    && $imagesDir/installPlatform.sh php $PHP81_VERSION \
+    && $imagesDir/installPlatform.sh php-composer $COMPOSER2_3_VERSION \
+    && cd /opt/php \
+    && ln -s $PHP81_VERSION 8 \
+    && ln -s $PHP81_VERSION lts \
+    && ln -sfn /opt/php/$PHP81_VERSION /home/codespace/.php/current \
+    && cd /opt/php-composer \
+    && ln -sfn $COMPOSER2_3_VERSION stable \
+    && ln -sfn /opt/php-composer/stable/composer.phar /opt/php-composer/composer.phar \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        libargon2-0 \
+        libonig-dev \
+    && rm -rf /var/lib/apt/lists/* \
+    && cp -f $imagesDir/build/benv.sh /opt/oryx/benv \
+    && cp -f $imagesDir/build/logger.sh /opt/oryx/logger \
+    && mkdir -p /usr/local/share/pip-cache/lib \
+    && chmod -R 777 /usr/local/share/pip-cache \
+    && ln -s /opt/buildscriptgen/GenerateBuildScript /opt/oryx/oryx \
+    && rm -f /etc/apt/sources.list.d/buster.list
+
+ENV ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/dotnet/lts:/opt/python/latest/bin:/opt/php/lts/bin:/opt/php-composer:/opt/yarn/stable/bin:/opt/hugo/lts::/opt/java/lts/bin:/opt/maven/lts/bin:/opt/ruby/lts/bin"
+
+ENV ORYX_PREFER_USER_INSTALLED_SDKS=true \
+    ORIGINAL_PATH="$PATH" \
+    PATH="$ORYX_PATHS:$PATH" \
+    CONDA_SCRIPT="/opt/conda/etc/profile.d/conda.sh" \
+    RUBY_HOME="/opt/ruby/lts" \
+    JAVA_HOME="/opt/java/lts" \
+    DYNAMIC_INSTALL_ROOT_DIR="/opt"
+
+# Now adding remaining of VSO platform features
+RUN buildDir="/opt/tmp/build" \
+    && imagesDir="/opt/tmp/images" \
+    # new next line
+    && add-apt-repository --remove ppa:xapienz/curl34 \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        apt-transport-https \
+        nano \
+        sed \
```

Comment:
nit: are the `sed` commands you added needed?

Reply:
Confirmed no longer needed